spec/models/item_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rails_helper'
 
 RSpec.describe Item, type: :model do
-  describe "relationships" do
-    it {should belong_to(:merchant)}
-  end
+  it { should belong_to(:merchant) }
```

Comment:
Why'd we get rid of the describe relationships block here? I think we should leave it for clarity

Reply:
We'll wait til we need validations etc before including "relationships" descriptions